app/src/main/java/fr/iut_rodez/pathpilot_android_client/model/Client.java

```diff
@@ -0,0 +1,253 @@
+package fr.iut_rodez.pathpilot_android_client.model;
+
+import android.content.Context;
+import android.os.Parcel;
+import android.os.Parcelable;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.List;
+
+import fr.iut_rodez.pathpilot_android_client.R;
+
+public class Client implements Parcelable {
```

Comment:
Add Javadoc class comments

Reply:
Fix in https://github.com/SAE-S5-01-Developpement-avance/PathPilot-android-client/pull/44/commits/f495a2fcc43e0fc3d934ae46d71b9af6a0856994